components/HoverLayer.svelte

```diff
@@ -36,20 +36,24 @@
       id: "hover-polygons",
       source: source,
       filter: isPolygon,
-      ...drawPolygon(hoverColor, 0.5),
+      // Outline around the polygons
+      // TODO Because this is underneath the draw controls, half the outline is
+      // covered by the 50% opaque color
+      ...drawLine(colors.hovering, lineWidth, 1.0),
     });
     $map.addLayer({
       id: "hover-lines",
       source: source,
       filter: isLine,
-      // TODO I'd like to cover up the base layers, but I can't figure out how to z-order on top of drawControls.
-      ...drawLine(hoverColor, 1.5 * lineWidth, 1.0),
+      // By "accident", this layer is underneath the draw controls. Draw a
```

Comment:
It's very tricky to z-order stuff from different Sources in maplibre. When you `addLayer`, you can tell it an ID of another layer to place under. But `mapbox-gl-draw` doesn't plumb through a way to do this, and I think the other layer has to exist first. There's also `moveLayer` for correcting things later. But figuring out when both layers are "ready" is tough, because it depends when the component gets mounted by Svelte and some asyncish thing finishes...